### DISCUSSION TITLE
Increased "First Load JS" Size After Adding Sentry to Next.js 13.4.10 Blueprint App

### DISCUSSION THREAD

**Discussion by Fateme-A** [vercel/next.js#52725] [Help]
### Summary

0

I have developed a Next.js blueprint app for testing purposes, using Next.js version 13.4.10. Recently, I integrated Sentry into the project to monitor errors and performance. However, I noticed a significant increase in the "First Load JS" size after incorporating Sentry. 
the Next.js build after adding sentry yo blueprint app:
`
Route (app)                                Size     First Load JS
┌ ○ /                                      4.97 kB         178 kB
└ ○ /favicon.ico                           0 B                0 B
+ First Load JS shared by all              173 kB
  ├ chunks/31aec8db-8b91092ba04b9594.js    43.4 kB
  ├ chunks/898-cb5c0042daf2e864.js         76.7 kB
  ├ chunks/bce60fc1-8eec208bad7cefab.js    50.6 kB
  ├ chunks/main-app-4b0256a570cf6ec2.js    558 B
  └ chunks/webpack-238fa72f00cf1a04.js     1.9 kB

Route (pages)                              Size     First Load JS
┌ ○ /404                                   226 B           154 kB
├ λ /api/sentry-example-api                0 B             154 kB
└ ○ /sentry-example-page (642 ms)          1.78 kB         155 kB
+ First Load JS shared by all              154 kB
  ├ chunks/framework-f34102d7a47a9b4a.js   45.3 kB
  ├ chunks/main-85542f7f347329ed.js        28.4 kB
  ├ chunks/pages/_app-5f34674dbf7c6dec.js  77.9 kB
  └ chunks/webpack-238fa72f00cf1a04.js     1.9 kB`
  I'm concerned about the increase in the "First Load JS" size and its potential impact on the application's performance. As a result, I would like to understand the reasons behind this increase and explore any possible optimizations or adjustments I can make to mitigate it. Specifically, I have the following questions: Is it expected to see an increase in the "First Load JS" size when incorporating Sentry into a Next.js 13.4.10 project? Are there any specific configuration options or best practices to optimize the bundle size when using Sentry in Next.js 13.4.10? Are there any particular dependencies or features of Sentry that might contribute to the increased "First Load JS" size?

Here's how I imported Sentry into my Next.js app:

const { withSentryConfig , init} = require("@sentry/nextjs");

**Comment by bscaspar** [comment]
Check out [this issue](https://github.com/getsentry/sentry-javascript/issues/2707) on the Sentry repo - long story short, they are working on it! It has a few suggestions for reducing package size based on usage. Ultimately though Sentry still contributes a significant amount to the initial load size.
